content.js

```diff
@@ -21,21 +22,57 @@ const translations = [
 	[" sister", "comrade"],
 	[" sis", "comrade"],
 	[" stranger", "comrade"],
-	["pal ", "comrade"],
-
+	[" pal ", "comrade"],
```

Comment:
This was actually intended because of PayPal, I'd like to keep it.

![image](https://github.com/doodlezucc/SovietWeb/assets/15276029/fc436b25-1191-4bbb-8986-d03e345f2688)

Reply:
it should work now

Reply:
I tried it already, it works now 👍